build/repo.props

```diff
@@ -0,0 +1,14 @@
+﻿<Project>
+  <PropertyGroup>
+    <DisablePackageReferenceRestrictions>true</DisablePackageReferenceRestrictions>
```

Comment:
This will go away when I move to ASP.NET Core 2.1 dependencies and add a dependencies.props file.